docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -194,6 +194,10 @@ Amplify.configure(awsExports);
 
 The latest version of the `Authenticator` component has several differences from earlier versions. Here are a few of the major changes that you'll need to look out for.
 
+<Alert variation="info" >
+  For previous use of `onAuthUiStateChange`, see [useAuthenticator](/react/connected-components/authenticator/headless#useauthenticator-hook).
+</Alert>
```

Comment:
What do you think about changing this to subheading `####` to match others here?